NOTICE.md

```diff
@@ -37,6 +37,10 @@ This project leverages the following third party content.
 See `DEPENDENCIES_FRONTEND` and `DEPENDENCIES_BACKEND` file. Further, the following third-party content is used that
 isn't listed in any DEPENDENCIES file:
 
+[Keycloak initial realm setup](./local/miw/keycloak-setup.json) derived from 
```

Comment:
But if we are not mentioning that third party content there, then the following sentence in the NOTICE.md wouldn't be correct anymore: 

"Further, the following third-party content is used that isn't listed in any DEPENDENCIES file:"

Reply:
Hm, at least that's what the experts from the EF told me. IMHO I would go with it. According to [ef handbook](https://www.eclipse.org/projects/handbook/#legaldoc-notice) your way would be kind of right. At least they refer only to libraries.

Problem with the overall approach is, that you specificy a path that may change over time and we miss it. Therfore I would go with the recommendation with the json.license. 

We're both right. I checked the recording of the office hour again (minute 57): "If you want to make more concrete attributions, you could also put it into the notice file. [...] But the more modern way it would be to put a .json.license file" 

Thus, I would like to go the propsed way with the .json.license for easier traceability.

Reply:
Alright. I have created a keycloak-setup.json.license file and that file now gets referenced in NOTICE.md file. Ok?